contracts/game/src/lib.cairo

```diff
@@ -224,6 +225,45 @@ mod Game {
             _start_game(ref self, weapon, name, interface_camel);
         }
 
+        fn optimistic_start(ref self: ContractState, adventurer_id: felt252, block_hash: felt252) {
```

Comment:
only the owner of the adventurer should be able to call this function, otherwise a bot could pwn every game of LS by calling this function on every new adventurer with an invalid

Reply:
Since the attack is getting called - it checks it in there